src/payment/strategies/stripev3/stripev3-payment-strategy.ts

```diff
@@ -0,0 +1,123 @@
+
+        if (!paymentMethod) {
+            throw new MissingDataError(MissingDataErrorType.MissingPaymentMethod);
+        }
+
+        return this._stripeScriptLoader.load(paymentMethod.initializationData.stripePublishableKey)
+            .then(stripeJs => {
+                this._stripeV3Client = stripeJs;
+                const elements = this._stripeV3Client.elements();
+                const cardElement = elements.create('card', {
+                    style: stripeOptions.style,
+                });
+
+                cardElement.mount(`#${stripeOptions.containerId}`);
+
+                this._cardElement = cardElement;
+
+                return Promise.resolve(this._store.getState());
+            });
+    }
+
+    execute(payload: OrderRequestBody, options?: PaymentRequestOptions): Promise<InternalCheckoutSelectors> {
+        const { payment, ...order } = payload;
+
+        if (!payment) {
+            throw new PaymentArgumentInvalidError(['payment']);
+        }
+
+        return this._store.dispatch(this._paymentMethodActionCreator.loadPaymentMethod(payment.methodId))
+            .then(state => {
+                const paymentMethod = state.paymentMethods.getPaymentMethod(payment.methodId);
+
+                if (!paymentMethod || !paymentMethod.clientToken) {
+                    throw new MissingDataError(MissingDataErrorType.MissingPaymentMethod);
+                }
+
+                return this._getStripeJs().handleCardPayment(
+                    paymentMethod.clientToken, this._getCardElement(), {}
+                ).then(stripeResponse => {
+                    if (stripeResponse.error || !stripeResponse.paymentIntent.id) {
+                        throw new StandardError(stripeResponse.error && stripeResponse.error.message);
+                    }
+
+                    const paymentPayload = {
+                        methodId: payment.methodId,
+                        paymentData: { nonce: stripeResponse.paymentIntent.id },
+                    };
+
+                    return this._store.dispatch(this._orderActionCreator.submitOrder(order, options))
+                        .then(() =>
+                            this._store.dispatch(this._paymentActionCreator.submitPayment(paymentPayload))
+                        );
+                });
+            });
+    }
+
+    finalize(options?: PaymentRequestOptions): Promise<InternalCheckoutSelectors> {
+        return Promise.reject(new OrderFinalizationNotRequiredError());
+    }
+
+    deinitialize(options?: PaymentRequestOptions): Promise<InternalCheckoutSelectors> {
+        this._getCardElement().unmount();
+
+        return Promise.resolve(this._store.getState());
+    }
+
+    private _getStripeJs(): StripeV3Client {
+        if (!this._stripeV3Client) {
+            throw new InvalidArgumentError('Unable to initialize payment because "stripeJs" argument is not provided.');
```

Comment:
You should throw `NotInitializedError` error here because the reason why it is undefined it's because `initialize` is not called.

PS: Side note, we actually don't recommend you to create private getters like this. This is because you want to throw different errors depending on the caller. i.e.: If you're trying to access the property in `execute`, you want to throw `NotInitializedError`, but if you're trying to access it in `initialize`, you'd want to throw a different error. I understand you try to avoid doing a check time you access the property. But the more important thing is to make sure we're throwing the correct error.